Babel modules not found

With the update to 1.1.0 building the app fails with 

```
ERROR` in ./node_modules/@nextcloud/vue/dist/Components/Multiselect.js
Module not found: Error: Can't resolve '@babel/runtime/helpers/asyncToGenerator' in '/builds/raimund.schluessler/Inventory/node_modules/@nextcloud/vue/dist/Components'
 @ ./node_modules/@nextcloud/vue/dist/Components/Multiselect.js 108:42-92
 @ ./node_modules/babel-loader/lib!./node_modules/vue-loader/lib??vue-loader-options!./src/components/RelationModal.vue?vue&type=script&lang=js&
 @ ./src/components/RelationModal.vue?vue&type=script&lang=js&
 @ ./src/components/RelationModal.vue
 @ ./node_modules/babel-loader/lib!./node_modules/vue-loader/lib??vue-loader-options!./src/components/TheItemDetails.vue?vue&type=script&lang=js&
 @ ./src/components/TheItemDetails.vue?vue&type=script&lang=js&
 @ ./src/components/TheItemDetails.vue
 @ ./src/components/TheRouter.js
 @ ./src/main.js

ERROR in ./node_modules/@nextcloud/vue/dist/Components/Multiselect.js
Module not found: Error: Can't resolve '@babel/runtime/helpers/typeof' in '/builds/raimund.schluessler/Inventory/node_modules/@nextcloud/vue/dist/Components'
 @ ./node_modules/@nextcloud/vue/dist/Components/Multiselect.js 108:8119-8159
 @ ./node_modules/babel-loader/lib!./node_modules/vue-loader/lib??vue-loader-options!./src/components/RelationModal.vue?vue&type=script&lang=js&
 @ ./src/components/RelationModal.vue?vue&type=script&lang=js&
 @ ./src/components/RelationModal.vue
 @ ./node_modules/babel-loader/lib!./node_modules/vue-loader/lib??vue-loader-options!./src/components/TheItemDetails.vue?vue&type=script&lang=js&
 @ ./src/components/TheItemDetails.vue?vue&type=script&lang=js&
 @ ./src/components/TheItemDetails.vue
 @ ./src/components/TheRouter.js
 @ ./src/main.js

ERROR in ./node_modules/@nextcloud/vue/dist/Components/Multiselect.js
Module not found: Error: Can't resolve '@babel/runtime/regenerator' in '/builds/raimund.schluessler/Inventory/node_modules/@nextcloud/vue/dist/Components'
 @ ./node_modules/@nextcloud/vue/dist/Components/Multiselect.js 87:41-78
 @ ./node_modules/babel-loader/lib!./node_modules/vue-loader/lib??vue-loader-options!./src/components/RelationModal.vue?vue&type=script&lang=js&
 @ ./src/components/RelationModal.vue?vue&type=script&lang=js&
 @ ./src/components/RelationModal.vue
 @ ./node_modules/babel-loader/lib!./node_modules/vue-loader/lib??vue-loader-options!./src/components/TheItemDetails.vue?vue&type=script&lang=js&
 @ ./src/components/TheItemDetails.vue?vue&type=script&lang=js&
 @ ./src/components/TheItemDetails.vue
 @ ./src/components/TheRouter.js
 @ ./src/main.js
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! inventory@0.0.1 build: `npm run sprites && webpack --progress --hide-modules --config webpack.prod.js`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the inventory@0.0.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

v1.0.0 built fine. Do you have any idea what might have changed?

When I remove `"babel-core": "^7.0.0-bridge.0",` I get a lot of these errors when I run `npm run test`:

```
Failed to collect coverage from /var/www/nextcloud/apps/tasks/src/components/DeleteCompletedModal.vue
ERROR: Cannot find module 'babel-core'
STACK: Error: Cannot find module 'babel-core'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/var/www/nextcloud/apps/tasks/node_modules/vue-jest/lib/compilers/babel-compiler.js:1:15)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
```


@georgehrke The same happens for the calendar app, if you *actually* do some testing. E.g. this test fails with the same error:

```javascript
import { shallowMount } from '@vue/test-utils'
import emptyCalendar from '../../../../src/components/EmptyCalendar.vue'

describe('components/emptyCalendar test suite', () => {

	it('should be true', () => {
		const wrapper = shallowMount(emptyCalendar)
		expect(true).toEqual(true)
	})

})
```

Your tests so far don't show this error, only because you don't actually test anything yet ;-)

--- a/babel.config.js
+++ b/babel.config.js
@@ -1,14 +1,10 @@
 module.exports = {
-	plugins: [
-		'@babel/plugin-transform-runtime',
-		'@babel/plugin-transform-react-jsx'
-	],
 	presets: [
 		[
 			'@babel/preset-env',
 			{
-				forceAllTransforms: true,
-				modules: false
+				useBuiltIns: 'usage',
+				corejs: 3
 			}
 		]
 	]

--- a/package.json
+++ b/package.json
@@ -31,9 +31,9 @@
 		"dist"
 	],
 	"dependencies": {
-		"@babel/polyfill": "^7.4.4",
 		"@nextcloud/axios": "^0.5.0",
 		"@nextcloud/router": "^0.1.0",
+		"core-js": "^3.4.4",
 		"escape-html": "^1.0.3",
 		"hammerjs": "^2.0.8",
 		"md5": "^2.2.1",
@@ -50,9 +50,6 @@
 	},
 	"devDependencies": {
 		"@babel/core": "^7.3.3",
-		"@babel/plugin-transform-arrow-functions": "^7.2.0",
-		"@babel/plugin-transform-react-jsx": "^7.3.0",
-		"@babel/plugin-transform-runtime": "^7.4.4",
 		"@babel/preset-env": "^7.3.1",
 		"@nextcloud/browserslist-config": "^1.0.0",
 		"babel-eslint": "^10.0.1",

--- a/webpack.common.js
+++ b/webpack.common.js
@@ -112,7 +112,6 @@ module.exports = {
 	],
 	resolve: {
 		alias: {
-			vue$: 'vue/dist/vue.esm.js',
 			Assets: path.resolve(__dirname, 'src/assets/'),
 			Components: path.resolve(__dirname, 'src/components/'),
 			Directives: path.resolve(__dirname, 'src/directives/'),
@@ -120,6 +119,7 @@ module.exports = {
 			Utils: path.resolve(__dirname, 'src/utils/'),
 			Fonts: path.resolve(__dirname, 'src/fonts/')
 		},
-		extensions: ['*', '.js', '.vue', '.json']
+		extensions: ['*', '.js', '.vue'],
+		symlinks: false
 	}
 };

